renderer/renderer.go

```diff
@@ -81,7 +83,21 @@ func (r *nodeRenderer) RegisterFuncs(reg goldmark.NodeRendererFuncRegisterer) {
 	reg.Register(east.KindTableCell, r.renderTableCell)
 }
 
+func (r *nodeRenderer) debugLog(entering bool, node ast.Node) {
+	if !r.conf.debug {
+		return
+	}
+	if entering {
+		fmt.Printf("%s<%s>\n", strings.Repeat("  ", r.debugPaddingLevel), node.Kind().String())
+		r.debugPaddingLevel += 1
+	} else {
+		r.debugPaddingLevel -= 1
+		fmt.Printf("%s</%s>\n", strings.Repeat("  ", r.debugPaddingLevel), node.Kind().String())
+	}
+}
```

Comment:
We use this as a library, can we make the log func a provided option instead? In particular we make page updates concurrently and this will really mess up the output if enabled as-is; a custom log func will let the output be e.g. collected until the end

Reply:
Made a suggestion over in https://github.com/sourcegraph/notionreposync/pull/15/files#r1603740342